lib/ansible/modules/cloud/scaleway/scaleway_compute.py

```diff
@@ -127,6 +127,12 @@
     - Time to wait before every attempt to check the state of the server
     required: false
     default: 3
+
+  security_group:
```

Comment:
If no value is provided, a default security group is automatically used, that might be mentioned here or in the `notes` section.